spec/models/course/experience_points_record_ability_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+RSpec.describe Course::ExperiencePointsRecord do
+  let!(:instance) { create(:instance) }
+  with_tenant(:instance) do
+    subject { Ability.new(user) }
+    let(:course) { create(:course) }
+    let(:course_student) { create(:course_student, :approved, course: course) }
+    let(:reason) { 'Reasonable reason' }
```

Comment:
is this necessary? the factory should make a valid record with some dummy reason.